tests/model_fields/tests.py

```diff
@@ -1023,3 +1024,74 @@ def get_prep_value(self, value):
             'exact', 'TEST', connection=connection, prepared=False
         )
         self.assertEqual(field.prep_value_count, 1)
+
+
+class ForeignObjectTest(unittest.TestCase):
```

Comment:
Subclass `django.test.TestCase` to maintain test isolation at the database level.